OpenCorePkg.dsc

```diff
@@ -78,6 +78,7 @@
   OcCpuLib|OpenCorePkg/Library/OcCpuLib/OcCpuLib.inf
   OcCryptoLib|OpenCorePkg/Library/OcCryptoLib/OcCryptoLib.inf
   OcDataHubLib|OpenCorePkg/Library/OcDataHubLib/OcDataHubLib.inf
+  OcDebugProtocolLib|OpenCorePkg/Library/OcDebugProtocolLib/OcDebugProtocolLib.inf
```

Comment:
No need for this line.

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/cc4b74b56bbc5a5a1d01a9c374a12137b563fedf

Reply:
It is required actually, otherwise it won't build: https://github.com/acidanthera/OpenCorePkg/runs/5375316040?check_suite_focus=true#step:6:405

Reply:
It just means you have mistakes in other places. Nothing should reference OcDebugProtocolLib.

Reply:
Indeed. Resolved in https://github.com/acidanthera/OpenCorePkg/pull/325/commits/fe42c6d1c620e92767573e8161239a6a810e1ff5. Thanks.